index.js

```diff
@@ -63,9 +64,11 @@ function plugin(opts) {
 
         debug('adding %s to includes as `%s`', resolvedFilename, name);
 
-        file[name] = included[filename].contents;
-
+        var fileClone = included[filename];
```

Comment:
you never used the the `clone` function - it should probably be something like:

```
file[name] = clone(included[filename])
```

and then there is no need to do `file[name].contents = `